local_store.go

```diff
@@ -23,15 +43,15 @@ func MemoryStore(meta map[string]json.RawMessage, files map[string][]byte) Local
 		meta:       meta,
 		stagedMeta: make(map[string]json.RawMessage),
 		files:      files,
-		signers:    make(map[string][]sign.Signer),
+		signers:    make(map[string][]*data.PrivateKey),
 	}
 }
 
 type memoryStore struct {
 	meta       map[string]json.RawMessage
 	stagedMeta map[string]json.RawMessage
 	files      map[string][]byte
-	signers    map[string][]sign.Signer
+	signers    map[string][]*data.PrivateKey
```

Comment:
i should make this consistent: and store the signers rather than the data like i do for filesystemstore